packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```
@@ -155,11 +162,6 @@ mkdir -p %{buildroot}%{_root_sbindir}
 mv %{buildroot}/%{gem_instdir}/sbin/upload-salt-reports %{buildroot}%{_root_sbindir}/upload-salt-reports
 mv .%{gem_instdir}/cron/smart_proxy_salt %{buildroot}%{_root_sysconfdir}/cron.d/%{gem_name}
 mkdir -p %{buildroot}%{smart_proxy_dynflow_bundlerd_dir}
-cat <<EOF | tee %{buildroot}%{smart_proxy_dynflow_bundlerd_dir}/smart_proxy_salt_core.rb \
-                %{buildroot}%{foreman_proxy_bundlerd_dir}/smart_proxy_salt_core.rb \
-                >/dev/null
-gem 'smart_proxy_salt_core'
-EOF
 
 %files
 %dir %{gem_instdir}
```

```suggestion
%dir %{gem_instdir}
%{_root_bindir}/salt_python_wrapper
```

or probably a line later (after `foreman-node`) so that it's sorted properly.

---

I've just created a [PR](https://github.com/theforeman/smart_proxy_salt/pull/73/files) to replace the `salt_python_wrapper` with a single line, given by an example from your blog. But I guess we'll need a new release then.